pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -200,7 +200,7 @@ func (l *podNICImpl) PlugPhase1(vmi *v1.VirtualMachineInstance, iface *v1.Interf
 			return createCriticalNetworkError(err)
 		}
 
-		err = bindMechanism.setCachedInterface(pidStr, iface.Name)
+		err = bindMechanism.setCachedInterface(fmt.Sprintf("%d", pid))
```

Comment:
better use `strconv.Itoa`

Reply:
Why ?

Reply:
Atoi is the specific function for the task, Sprintf is a generic pattern rendering mechanism that happened do what you need, I am not concerned about performance here, more about readability.

Reply:
Throughout this entire pkg this is how an int gets converted to a string:
  - https://github.com/kubevirt/kubevirt/blob/2122c2beb050399c79a290f880c3fac7295a18ea/pkg/virt-launcher/virtwrap/network/podinterface.go#L179
  - https://github.com/kubevirt/kubevirt/blob/2122c2beb050399c79a290f880c3fac7295a18ea/pkg/virt-launcher/virtwrap/network/common.go#L445
  - https://github.com/kubevirt/kubevirt/blob/2122c2beb050399c79a290f880c3fac7295a18ea/pkg/virt-launcher/virtwrap/network/common.go#L446

I actually do not care much about how to do it, I personally find both readable, but I'd prefer to be consistent at pkg level. That can be done in an assorted follow up PR.